docs/source/developer_guide/development-workflow.md

```diff
@@ -153,7 +153,7 @@ not just refresh your browser.
 Elyra's container image can be built using:
 
 ```bash
-make container-image
+make elyra-image
```

Comment:
Please update the output of `make` above.  Since this target is the ONLY target documented, I believe it should also be included in the `make` command help string - along with the two other targets.

Reply:
I'd rater remove the output because it'll be likely outdated again in the future and keeping it in synch just extra maintenance work we don't need.

Reply:
I would prefer to update it, and maybe clean up some of the more internal dev targets (e.g. leave install and hide install-server install-ui)

Reply:
Okay. Then I'll put this on hold until the `Makefile` is cleaned up.

Reply:
To speed up things you could also add a commit with a cleaned-up version of the `Makefile`. I don't really have a horse in the race (as to what is internal vs external) because I'm pretty much only using two targets regularly.

Reply:
Resolved by https://github.com/elyra-ai/elyra/pull/1553